openedx/features/redhouse_features/registration/migrations/0001_initial.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-07-27 06:53
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='AdditionalRegistrationFields',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('user_type', models.CharField(choices=[(b'student', b'Student'), (b'staff', b'Staff')], max_length=8, verbose_name=b'User Type')),
+                ('phone', models.CharField(blank=True, max_length=16, verbose_name=b'Phone')),
+                ('sch_org', models.CharField(blank=True, max_length=150)),
+                ('organization_type', models.CharField(choices=[(b'corporate', b'Corporate'), (b'school district', b'School District'), (b'private school', b'Private School'), (b'higher ed', b'Higher ED'), (b'training', b'Training')], max_length=32, verbose_name=b'Organization Type')),
+                ('country', models.CharField(max_length=32, verbose_name=b'Country')),
+                ('user', models.OneToOneField(null=True, on_delete=django.db.models.deletion.SET_NULL, to=settings.AUTH_USER_MODEL)),
```

Comment:
nit:
```suggestion
                ('user', models.OneToOneField(null=True, on_delete=models.deletion.SET_NULL, to=settings.AUTH_USER_MODEL)),
```
this way you can remove the import above.